.github/actions/devcontainer_run_command/action.yml

```diff
@@ -164,7 +164,7 @@ runs:
       shell: bash
       run: |
         # Write command to a command.sh script file
-        cat <<EOF >> ./command.sh
+        cat <<EOF > ./command.sh
```

Comment:
Why is this different to upstream? 

https://github.com/microsoft/AzureTRE/blob/3fcbdf9c6df25ccd3500702bff952e95766fa6d2/.github/actions/devcontainer_run_command/action.yml#L182

Reply:
It should be like this in the upstream as well.

Reply:
Can we PR this upstream then?